### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by fedyiv**

### CODE REVIEW

File: Specs/ssi_agent/0.0.1/ssi_agent.podspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+    s.name         = "ssi_agent"
+    s.version      = "0.0.1"
+    s.summary      = "Kotlin Multiplatform"
+    s.description  = "Self Sovereign Identity Kotlin Multiplatform Agent.A new Mobile SDK to use with Self Sovereign Identity (SSI) applications is available as an open-source project from Luxoft DXC. The SSI Mobile SDK provides libraries and tools to accelerate integration of digital wallet capabilities into existing mobile apps."
+    s.homepage     = "https://kotlinlang.org/docs/multiplatform.html"
+    s.license      = "All the original code is licensed under the Apache 2.0 License."
+    s.author       = { "Krzysztof Kamyczek" => "krzysztof.kamyczek@dxc.com" }
+    s.source       = { :http => "https://github.com/kkamyczek/ssi-mobile-sdk/releases/download/v1.0/ssi_agent.xcframework.zip"}
```
Comment (kkamyczek) [inline Specs/ssi_agent/0.0.1/ssi_agent.podspec:9]: Only this change. We can change it to the Azure folder. However, we need to test if there is a zip or just a folder.

Comment (kkamyczek) [reply Specs/ssi_agent/0.0.1/ssi_agent.podspec:9]: Here is a path: https://github.com/Lumedic/ssi-mobile-sdk/releases/download/0.0.1/dxc-ssi-agent-ios-0.0.1.zip

Comment (fedyiv) [reply Specs/ssi_agent/0.0.1/ssi_agent.podspec:9]: So, you suggest to wait until we merge the latest change from you, right?

Comment (kkamyczek) [reply Specs/ssi_agent/0.0.1/ssi_agent.podspec:9]: yes, please

**Review by kkamyczek — APPROVED** [review]

**Pull request merged**